bin/ci

```diff
@@ -147,10 +147,9 @@ EOF
 prepare_build() {
   on_linux verify_linux_environment
 
-  on_linux docker pull "jhass/crystal-build-$ARCH"
+  on_linux docker pull "jhass/crystal-build-$ARCH:llvm38"
 
-  on_osx brew install libevent crystal-lang
-  on_osx /bin/bash -c "'curl \"http://crystal-lang.s3.amazonaws.com/llvm/llvm-3.5.0-1-darwin-x86_64.tar.gz\" | tar xz -C /tmp'"
+  on_osx brew install llvm libevent crystal-lang
```

Comment:
The llvm formula seems to be at 3.6, see my overview comment above. There's a formula for 3.8 in a separate repo, but either builds LLVM from source, which takes forever and is just not acceptable. Also just deleting the `on_osx` below is wrong.